services/harmony/test/ignore-errors.ts

```diff
@@ -15,14 +15,11 @@ import { resetQueues } from './helpers/queue';
 import hookServersStartStop from './helpers/servers';
 import { fakeServiceStacOutput, getWorkForService, updateWorkItem } from './helpers/work-items';
 
-const reprojectAndZarrQuery = {
+const hossAndMaskfillQuery = {
   maxResults: 1,
-  outputCrs: 'EPSG:4326',
-  interpolation: 'near',
-  scaleExtent: '0,2500000.3,1500000,3300000',
-  scaleSize: '1.1,2',
-  format: 'application/x-zarr',
-  concatenate: false,
+  subset: 'lat(60:65)',
+  format: 'application/x-netcdf4',
+  forceAsync: true,
```

Comment:
I had to add `forceAsync: true` to the HOSS/MaskFill requests in a couple of places, as otherwise the condition for `numInputs <= maxSyncGranules` was letting Harmony try and do a synchronous request and that was timing out.

This feels like the simplest way to not end up doing that (without changing the `services-uat.yml` file), but definitely open to other ideas.